localization/yabloc/yabloc_pose_initializer/src/semantic_segmentation/semantic_segmentation_server.py

```diff
@@ -31,26 +38,43 @@ def __init__(self):
         model_path = self.declare_parameter("model_path", "").value
 
         self.get_logger().info("model path: " + model_path)
-        self.dnn_ = core.SemanticSegmentationCore(model_path)
         self.bridge_ = CvBridge()
 
+        if os.path.exists(model_path):
+            self.dnn_ = core.SemanticSegmentationCore(model_path)
+        else:
+            self.dnn_ = None
+            self.__print_error_message()
+
         self.srv = self.create_service(
             SemanticSegmentation, "semantic_segmentation_srv", self.on_service
         )
 
+    def __print_error_message(self):
+        messages = error_message.split("\n")
```

Comment:
```suggestion
        messages = ERROR_MESSAGE.split("\n")
```

Reply:
OK :ok_hand:  I have already fixed it in c809a20